tests/test_typing_extensions.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pathlib
+import re
+
+from absl.testing import absltest
+
+TYPING_RE = re.compile(r'\btyping\b.*TypedDict')
+
+class TypingExtensionsTests(absltest.TestCase):
+    def test_no_typing_typed_dict(self):
```

Comment:
Is there an exit-condition for this lint rule? If so we should document it here somewhere.

Reply:
Done.